### PR TITLE
Feature: Bulk action to anonymize participant profiles

### PR DESCRIPTION
**Pull request by bgreiner**
This feature adds the possibility to anonymize a subset or all of the participant profiles resulting from a participant query in one go. To do so, go to Participants -> Edit active/all participants -> Search, and then select "do ..."-> Anonymize profiles. You can choose to transfer the anonymized profiles to a different participant status, e.g. "Anonymized". To configure which participant profile fields should be anonymized and to which dummy values they should be set, go to Options/Participant profile/Fields to anonymize in anonymization bulk action.

Technically, this new feature uses and extends existing functionality to edit (ordered) lists (configuration) and the bulk action infrastructure (execution).

This feature makes use of 2 new user privileges
participants_bulk_anonymization
pform_anonymization_fields_edit
which need to be defined for the respective user types.

It also makes use of a number of new language terms that need to be defined:
fields_to_anonymize_in_anonymization_bulk_action
anonymized_dummy_value
anonymize_profiles
anonymize_profiles_for
fields_will_be_anonymized_as_follows
disclaimer_anonymize_profiles
upon_anonymization_change_status_to
profile_anonymize
error_no_fields_to_anonymize_defined
xxx_participant_profiles_were_anonymized

### CODE REVIEW

File: config/dbupdates.php [hunk 1]
```diff
@@ -40,4 +40,123 @@
 */
 
 
+$system__database_upgrades[]=array(
+'version'=>'2020021200',
```
Comment (kabalin) [inline config/dbupdates.php]: May be it worth indenting all items in array? i.e
```
$system__database_upgrades[]=array(
    'version'=>'2020021200', 
    'type'=>'new_admin_right',
    'specs'=> array(
        'right_name'=>'pform_anonymization_fields_edit', 
        'admin_types'=>array('admin','developer','installer'),
    )
);
```

File: tagsets/options.php [hunk 2]
```diff
@@ -257,12 +257,14 @@ function options__ordered_lists_get_current($poss_cols,$saved_cols,$sortby_radio
...
         if (!isset($arr['cols'])) $arr['cols']='<TD>'.$arr['display_text'].'</TD>';
-        if ($sortby_radio) {
+        if ($extra_field && $extra_field=='sortby_radio') {
             if ($arr['sortable']) {
                 $arr['cols'].='<TD><INPUT type="radio" name="sortby" value="'.$k.'"';
                 if (isset($arr['item_details']['default_sortby']) && $arr['item_details']['default_sortby']) $arr['cols'].=' CHECKED';
                 $arr['cols'].='></TD>';
             } else $arr['cols'].='<TD></TD>';
+        } elseif ($extra_field && $extra_field=='field_value') {
+                $arr['cols'].='<TD><INPUT type="text" size="30" maxlength="255" name="field_values['.$k.']" value="'.$arr['item_details']['field_value'].'"></TD>';
```
Comment (kabalin) [inline tagsets/options.php]: double indent here

File: config/dbupdates.php (hunk 1 above)
Comment (bgreiner) [reply config/dbupdates.php]: Ok, done.

File: tagsets/options.php (hunk 2 above)
Comment (bgreiner) [reply tagsets/options.php]: Ok, done.

**Pull request merged**